### PR TITLE
Update main.py

### PR DESCRIPTION
**Pull request by NILL69**
Now pupils can see 
1) a schedule of lessons and food (from canteen); 
2) an old homework (users asked to add it because there can be a situation when someone missed a lesson but he still wants to solve homework that was assigned for this lesson)

Now administrators and teachers can add homework easily due to there are buttons with school subjects (early they had to write a school subject by their own)

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -202,7 +326,7 @@ def _gen_cmd_add_homework():
 # Homework command
 def _cmd_homework(_tb, _message, _args):
     now = datetime.datetime.now().timestamp()
-    hw_list = database.get_hw_since(now - 60 * 60 * 24)
+    hw_list = database.get_hw_since(now - 60 * 60 * 24 * 7)
```
Comment (NILL69) [inline src/main.py:335]: now homework will be deleted only after the week passes from homework deadline

File: src/main.py [hunk 2]
```diff
@@ -173,6 +290,13 @@ def _cmd_add_homework(_tb, _message, args):
     # )
     return "Задание добавлено 📚", None
 
+def gen_homework_menu():
+    """Generates menu for homework"""
+    return Keyboard(
+        [
+            for i in subjects
```
Comment (NILL69) [inline src/main.py]: function to create buttons that contain school subjects

File: src/main.py (hunk 1 above)
Comment (maksimil) [reply src/main.py:335]: It's not ever **deleted**... It's not **shown**

File: src/main.py [hunk 3]
```diff
@@ -21,9 +21,112 @@
...
+2 неделя
+🍑Понедельник: овощной суп со свежей капустой, рис с куриной котлетой в сыре, оливье без соуса школьный, сок яблочный, яблоко
+🍑Вторник: рыбный суп, рагу, салат из капусты яблока и морковки, йогурт
+🍑Среда: борщ с фасолью, гречка с котлетой вкусной, булка с творогом, полпомидора
+🍑Четверг: рассольник, рыба в яйце с пюре, свекла, йогурт
+🍑Пятница: похлебка крестьянская, курица в сперме, морковка, яблоко
+🍑Суббота: суп с картошкой, тушеные овощи с куриной котлетой, маринованный огурец, яблочный сок, яблоко
+""" 
+
+subjects = {
```
Comment (maksimil) [inline src/main.py]: `subjects` also capsed

File: src/main.py [hunk 4]
```diff
@@ -261,12 +385,23 @@ def _interface(_tb, chatid):
...
             ]
         )
     else:
-        return _build_interface([CMD_CALENDAR, CMD_HOMEWORK, CMD_START, CMD_HELP])
+        return _build_interface(
+            [
+                CMD_CALENDAR, 
+                CMD_HOMEWORK, 
+                _cmd_lessons_schedule, 
+                _cmd_foodCanteen_schedule,
```
Comment (maksimil) [inline src/main.py]: I assume you haven't tested this, bc these are functions

File: src/main.py [hunk 5]
```diff
@@ -173,6 +290,13 @@ def _cmd_add_homework(_tb, _message, args):
     # )
     return "Задание добавлено 📚", None
 
+def gen_homework_menu():
```
Comment (maksimil) [inline src/main.py]: No reason for this to be a function. It returns the same on every invocation. It's just a constant

File: src/main.py [hunk 6]
```diff
@@ -56,6 +159,20 @@ def _cmd_help(tb, message, _args):
...
 
+# Schedule lesson command
+def _cmd_lessons_schedule():
+    return (
+        LESSONS_SCHEDULE,
+        None   
+    )
+
+# Schedule food_canteen command
+def _cmd_foodCanteen_schedule():
```
Comment (maksimil) [inline src/main.py]: Mixing *camelCase* with *snake_case*

File: src/main.py (hunk 3 above)
Comment (maksimil) [reply src/main.py]: and there is a better a way to list subjects, we need to discuss this

File: src/main.py (hunk 1 above)
Comment (maksimil) [reply src/main.py:335]: In any case, this is probably a good idea to test

File: src/main.py (hunk 4 above)
Comment (maksimil) [reply src/main.py]: There is a way to test all of this without `docker` btw

File: src/main.py [hunk 7]
```diff
@@ -21,9 +21,112 @@
 
 Всё будет приходить прямо в этот чат
 
-👇Нажте кнопку \"Календарь\" и увидишь, что запланировано на ближайшее будущее"
+👇Нажмите кнопку \"Календарь\", чтобы увидеть, что запланировано на ближайшее будущее"
 """
 
+LESSONS_SCHEDULE = """
```
Comment (maksimil) [inline src/main.py:27]: Idk about this one

File: src/main.py (hunk 2 above)
Comment (maksimil) [reply src/main.py]: If you think that you need to comment your code, please do it in code, not in PRs

**Review by maksimil — CHANGES_REQUESTED** [review]

File: src/main.py [hunk 8]
```diff
@@ -290,13 +290,7 @@ def _cmd_add_homework(_tb, _message, args):
...
     return "Задание добавлено 📚", None
 
-def gen_homework_menu():
-    """Generates menu for homework"""
-    return Keyboard(
-        [
-            for i in subjects
-        ]
-    )
+SUBJECTS_MENU = Keyboard([for i in SUBJECTS])
```
Comment (maksimil) [inline src/main.py]: `Keyboard` asks for a `[[str]]`, but you  are providing `[str]`.

Just put this
```python
SUBJECTS_MENU = Keyboard([
    ["Русский", "Литература"],
    ["Алгебра", "Геометрия"]
])
```

without a `SUBJECTS` variable

**Review by maksimil — CHANGES_REQUESTED** [review]

**Review by maksimil — APPROVED** [review]

**Pull request merged**